script.js

```diff
@@ -0,0 +1,61 @@
+let bookCollection = {};
+let booksArray = Object.keys(bookCollection);
+
+function storeBooks() {
+  window.localStorage.setItem('books', JSON.stringify(bookCollection));
+}
+
+function createList() {
+  booksArray = Object.keys(bookCollection);
+  booksArray.forEach((book) => {
+    const removeButton = document.createElement('button');
+    const dividerLine = document.createElement('hr');
+
+    removeButton.innerHTML = 'Remove';
+    removeButton.id = book;
+
+    const bookSection = document.createElement('div');
+    const title = document.createElement('p');
+    const author = document.createElement('p');
+
+    title.innerHTML = book;
+    author.innerHTML = bookCollection[book];
```

Comment:
`innerHtml` is dangerous here because it takes user input and parses it as HTML. I can put a script in there and run whatever I want.

For example, I was able to make an element `h1`:
<img width="201" alt="image" src="https://user-images.githubusercontent.com/34813339/168485113-74edde02-a528-4376-9ad3-addd96ea7c5f.png">